README.md

```diff
@@ -1,49 +1,57 @@
 # Style Guide
+
+## Tables of Contents
+> ## Naming Conventions
+> ## Comments
+> ## IPC Colors
+
 ## Naming Conventions
-**snake_case**: all words are lowercase and are seperated by '_'.  
-**html-snake-case**: all words are lowercase and are seperated by '-'.  
-**lowerCamelCase**: starts with a lowercase letter and following words start with uppercase letters with no spaces between words.  
-**UpperCamelCase**: all words start with capital letters with no spaces between words.  
-**lower spaced out**: all words are lowercase and normally spaced.  
-**Upper Spaced Out**: all words start with an uppercase letters and are normally spaced.  
+- Variables : snake_case : all words are lowercase and are separated by '_'
+- HTML Atrributes : html-snake-case : all words are lowercase and are separated by '-' 
+- Functions : lowerCamelCase : starts with a lowercase letter and following words start with uppercase letters with no spaces between words
+- Tables :  UpperCamelCase : all words start with capital letters with no spaces between words
+- Table Fields : lower spaced out : all words are lowercase and normally spaced
+- Script Includes : Upper Spaced Out : all words start with an uppercase letter and are normally spaced
```

Comment:
The definition of naming conventions shouldn't be tied to items straight away since there might be two items that share a naming convention.